### DISCUSSION TITLE
Minimum order fee

### DISCUSSION THREAD

**Discussion by centoasa** [PrestaShop/PrestaShop#40203] [Ideas]
Prestashop 1.7.5.1 on debian
I believe it is a great feature to add to the minimum order for amount, even a minimum order surcharge.
That is, set an amount below it the customer must pay a supplement. Example: orders below €50, fees: €5
I've search a lot on google but there's nothing about it.
Thank you

---
> Originally posted by @margud

#### Is your feature request related to a problem?

More and more merchants want to add payment fees for their payment methods but now it's really hard to do it even with modules. We need to do all kinds of workarounds for it to work.

#### Describe the solution you'd like

We want PrestaShop to add an option for the payment method to have fees just like carriers do.
The fee should be displayed next to the payment method and when selected it should be added to the total price.

#### Additional context

![image](https://user-images.githubusercontent.com/51944825/78526304-4f00fb00-77e2-11ea-8bac-8c1441a51650.png)

**Comment by khouloudbelguith** [comment]
@marionf, @colinegin what do you think of his improvement?
Thanks!

**Comment by ghost** [comment]
Hi @centoasa ,

We need more info to better understand your need.
In which case do you want to apply this supplement? What is the goal of this supplement? (boost sales, increase mean basket value, easier stock management, ...)

**Comment by centoasa** [comment]
it's very simple: some suppliers apply this tax on the minimum order, so I have to transfer this amount to the customer. It would be easy to increase the selling price, but on search engines and price comparators you would be very high. Instead, by applying an additional cost below a minimum amount, the customer could be encouraged to increase the quantity purchased. That is if I have to pay 5 euros as a supplement, I prefer to add 1pcs to my cart, pass the minimum order and for the same total I have an extra product.

**Comment by ghost** [comment]
Ok, I understand.

It is possible to set a "Minimum purchase total required in order to validate the order"
It is not exactly what you are looking for but combined with a banner it could work.

I also found a module on the Addons Marketplace that does exactly what you are looking for (check PS version compatibility) :
https://addons.prestashop.com/en/registration-ordering-process/29149-small-quantity-surcharge.html


We'll look into adding this feature natively, though we prefer to focus on features not solved by modules or config options combination.

**Comment by ghost** [comment]
"Minimum purchase total required in order to validate the order" is accessible in Shop Parameters>Order Settings

![Capture d’écran 2019-04-09 à 14 53 15 (2)](https://user-images.githubusercontent.com/47456895/55804654-0c2c1a80-5add-11e9-9ca7-4a208efa14c6.png)

**Comment by centoasa** [comment]
Thanks for your time @samuel-pires 
Mine was a suggestion for future.

**Comment by ghost** [comment]
Hi @centoasa,
I shared this idea on feedbak.prestashop.com to get some feedback from the community.
http://feedback.prestashop.com/forums/387864-prestashop-1-7-x/suggestions/37672468-option-to-add-a-fee-if-total-order-value-is-below

Best,

**Comment by jitendra-cygnet** [comment]
Hi @samuel-pires  I have redirect my cart page to checkout as i have created custom one page checkout. 

Now issue is, its not checking minimum order value in my checkout page. i am able to do order.

Prestashop 1.7

I have hide default redirection of cart page as i am not using it
